src/styles/global/_typography.scss

```diff
@@ -0,0 +1,29 @@
+/*============= Helpers ================*/
+
+.ta-center {
+  text-align: center;
+}
+
+.tt-uppercase {
+  text-transform: uppercase;
+}
+
+.tc-white {
+  color: $white;
+}
+
+.tc-black {
+  color: $black;
+}
+
+
+/*============= Elements ================*/
+
+h1 {
```

Comment:
could we assign this to a class name like .h1? it is likely that there will be instances where the structural h-level will not directly map to type hierarchy.

i also noticed that we should set the font-weight on all header elements to normal—the browser is automatically applying a bold style but actually all the type is supposed to be normal weight.

Reply:
Yep! I will change the markup on this. If you want, I can add a stylesheet to reset all of the default browser styles?

Reply:
:)